### PR TITLE
dbeaver/pro#2555 Get driver lists for different DBeaver Editions

### PR DESCRIPTION
**Pull request by serjiokov**
Specific service provided to export drivers (databases) list to JSON file.

### CODE REVIEW

**Comment by serjiokov** [comment]
Flag requires to use: -`databaseLog`

**Comment by serjiokov** [comment]
[supported.databases.json](https://github.com/dbeaver/dbeaver/files/14670877/supported.databases.json)

**Review by daelynum — APPROVED** [review]

**Comment by serjiokov** [comment]
Uasge: -databaseList D:\Path\To\Folder

**Comment by serjiokov** [comment]
List of databases in JSON format
[database.drivers.json](https://github.com/dbeaver/dbeaver/files/14693375/database.drivers.json)

File: plugins/org.jkiss.dbeaver.ui.app.standalone/plugin.xml [hunk 1]
```diff
@@ -471,6 +471,14 @@
             hasArg="true"
             handler="org.jkiss.dbeaver.ui.app.standalone.cli.SQLTranslatorHandler"/>
     </extension>
+    <extension point="org.jkiss.dbeaver.commandLine">
```
Comment (serge-rider) [inline plugins/org.jkiss.dbeaver.ui.app.standalone/plugin.xml]: Why duplicate <extension point="org.jkiss.dbeaver.commandLine"> ?

File: plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/DBeaverCommandLine.java [hunk 2]
```diff
@@ -71,6 +72,7 @@ public class DBeaverCommandLine
     public final static Options ALL_OPTIONS = new Options()
         .addOption(PARAM_HELP, PARAM_HELP, false, "Help")
         .addOption(PARAM_CONFIG, "variablesFile", true, "Uses a specified configuration file for variable resolving")
+        .addOption(PARAM_DB_LIST, "database-driver-list", true, "Store list of supported database drivers in json format")
```
Comment (serge-rider) [inline plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/DBeaverCommandLine.java]: Show list of supported dtabase drivers.

**Review by serge-rider — CHANGES_REQUESTED** [review]

File: plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/DBeaverCommandLine.java (hunk 2 above)
Comment (serjiokov) [reply plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/DBeaverCommandLine.java]: Done

**Comment by serjiokov** [comment]
List of drivers in JSON format
[database.drivers.json](https://github.com/dbeaver/dbeaver/files/14696043/database.drivers.json)

**Review by LonwoLonwo — APPROVED** [review]

**Comment by Matvey16** [comment]
Verified

**Pull request merged**